src/search/SearchMessagesScreen.js

```diff
@@ -18,7 +18,7 @@ import { fetchMessages } from '../message/fetchActions';
 type OuterProps = $ReadOnly<{|
   // These should be passed from React Navigation
   navigation: AppNavigationProp<'search-messages'>,
-  route: RouteProp<'search-messages', void>,
+  route: RouteProp<'search-messages', {| streamId: number |}>,
```

Comment:
This type needs to match up with what's passed to it from `navActions`. See previous discussion: https://github.com/zulip/zulip-mobile/pull/5189#discussion_r781790191